test/field-collection.test.js

```diff
@@ -24,18 +24,20 @@ describe('FieldCollection', function() {
     assert.equal(doc.fields.get('foo').parent, doc);
   });
 
-  it('should trigger change:probability events in unaffected children', function(done) {
+  it('should NOT trigger change:probability events in unaffected children', function() {
```

Comment:
I don't know if your change is correct. "Unaffected" may be the wrong word, because the probability does change for Number type in this example. This can cause strange bugs.